spec/features/merchants/dashboard_spec.rb

```diff
@@ -4,13 +4,52 @@
   describe 'as a merchant' do
     let!(:merchant_1) { Merchant.create!(name: "Johns Tools") }
     let!(:merchant_2) { Merchant.create!(name: "Hannas Hammocks") }
-    describe 'When I visit my merchant dashboard' do
+    
+    describe 'User Story 1' do
```

Comment:
I like adding the user story to the describe block.  We may want to refactor it as a merchant user story 1, to differentiate from admin user stories.